### ISSUE TITLE
`list` not showing all resources in a container

### ISSUE THREAD

**Issue by jeswr** [SolidLabResearch/Bashlib#42]
I have created [this minimal repro](https://github.com/jeswr/bashlib-list-error-minimal-repro) which:
1. Inserts a `contacts` resource into the root of a Pod storage
2. Calls `list` on that root

As seen in the repro `list` does not show `contacts` when I expect that it would.

**Comment by jeswr** [comment]
Calling `list` is also not showing the existing `README` or `profile/` resources in the root.

I suggest that these be discovered from the containment triples of the container

```ttl
@prefix dc: <http://purl.org/dc/terms/>.
@prefix ldp: <http://www.w3.org/ns/ldp#>.
@prefix posix: <http://www.w3.org/ns/posix/stat#>.
@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.

<> a <http://www.w3.org/ns/pim/space#Storage>, ldp:Container, ldp:BasicContainer, ldp:Resource;
    dc:modified "2022-12-29T04:59:32.000Z"^^xsd:dateTime;
    ldp:contains <README>, <profile/>, <contacts>.
```

**Comment by Dexagod** [comment]
Hi Jesse,

Found the issue:
The problem was that the code is trying to look for additional metadata for these `contains` urls, which are not available here (probably are not mandatory for the Solid spec).
So instead of just returning the URIs it returned nothing.

Fixed with 797f099a6c4a7c7e70b50bfd3e9bd9d450a5b3d9
